### PR TITLE
Start setting up a small parameter model that can be used by serial

### LINKED ISSUE #98: Serial: set up parameters
Create a small parameter model to test the serial UI/panda plans. Where possible, create smaller components that can be reused by other parameter models

### PR DESCRIPTION
**Pull request by noemifrisina**
Closes #98

### CODE REVIEW

File: src/i19_bluesky/parameters/serial_parameters.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    wells: WellsSelection
+    # Missing: detector_name, pinhole_size, sample_stage, also axes values
+    rot_axis_start: float
+    rot_axis_increment: float
+    rotation_axis: RotationAxis = RotationAxis.PHI
+    # The other positions can be read from device for now and then set to detector
+
+    @computed_field
+    @property
+    def tot_num_images(self) -> int:
```
Comment (shree-iyengar-dls) [inline src/i19_bluesky/parameters/serial_parameters.py]: Nit: I'd say total instead of tot 
```suggestion
    def total_num_images(self) -> int:
```

File: tests/unit_tests/parameters/test_components.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    model = {
+        "rotation_axis": "phi",
+        "scan_start_deg": -10,
+        "scan_increment_deg": 0.2,
+        "scan_steps": 10,
+        "exposure_time_s": 0.1,
+    }
+    params = ZebraRotationParams(**model)
+
+    assert params.rotation_direction == "Positive"
```
Comment (shree-iyengar-dls) [inline tests/unit_tests/parameters/test_components.py]: Could: Does there need to be a test to check that it can be set to 'Negative' since 'Positive' is the default?

File: tests/unit_tests/parameters/test_serial_parameters.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        "two_theta_deg": 0,
+        "transmission_fraction": 0.3,
+        "grid": {
+            "grid_type": "Silicon",
+            "x_steps": 20,
+            "z_steps": 20,
+        },
+        "wells": dummy_wells_settings,
+        "rot_axis_start": -5,
+        "rot_axis_increment": 0.1,
```
Comment (shree-iyengar-dls) [inline tests/unit_tests/parameters/test_serial_parameters.py]: Could: Have 'rotation' instead of 'rot' just for clarity
```suggestion
        "rotation_axis_start": -5,
        "rotation_axis_increment": 0.1,
```

**Review by shree-iyengar-dls — CHANGES_REQUESTED** [review]
Most of my comments are just small suggestions for renaming things tbh

File: src/i19_bluesky/parameters/serial_parameters.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        return self.images_per_well * self.wells.num_wells_to_collect
+
+    # @property
+    # @abstractmethod
+    # def detector_params(self): ...
+    # TODO see https://github.com/DiamondLightSource/i19-bluesky/issues/103
+
+
+class SerialExperimentEh2(SerialExperiment):
```
[the remark this answers is not in the document]
Comment (shree-iyengar-dls) [reply src/i19_bluesky/parameters/serial_parameters.py]: Ohh sorry didn't know about this
